src/graphql/api.ts

```diff
@@ -79,9 +81,39 @@ export class GraphQLAPI {
     // };
 
     search = async (name: string): Promise<SearchResult[]> => {
-        const result = tracks.filter((track) => track.title === name);
+        const artists1 = artists.filter((artist) => artist.name === name);
```

Comment:
All search queries should be a substrings of results, not exactly result (For example, if you type 'Trav' server should response with Travis Scott, now it responses with empty array result)